lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/dataaccess/ResearchProjectDAO.java

```diff
@@ -33,17 +33,9 @@
 	public ResearchProject update(ResearchProject toUpdate) throws NotFoundException;
 
 	/**
-	 * Update the ownership of an existing ResearchProject.
+	 * Delete a ResearchProject given its ID.
 	 * 
-	 * @param researchProjectId
-	 * @param newOwnerId
-	 * @return
-	 * @throws NotFoundException if the ResearchProject cannot be found.
-	 */
-	public ResearchProject changeOwnership(String researchProjectId, String newOwnerId) throws NotFoundException;
```

Comment:
I prefer an update of only the fields that have changed as apposed to updating the entire row.